src/record.js

```diff
@@ -123,26 +123,17 @@ export default class Record extends Feature {
     json.updated_by_id = this._updatedByID || null;
     json.updated_by = this._updatedBy || null;
 
-    if (this._horizontalAccuracy != null) {
-      json.horizontal_accuracy = this._horizontalAccuracy;
-    }
-
-    if (this._verticalAccuracy != null) {
-      json.vertical_accuracy = this._verticalAccuracy;
-    }
-
+    /* eslint-disable no-undefined */
+    json.horizontal_accuracy = this._horizontalAccuracy === undefined ? null : this._horizontalAccuracy;
```

Comment:
Oh yeah . . . I forgot about that operator. That's definitely cleaner, but messing around with babel is a bit more than I want to tackle in this PR.

My initial intent was to get rid of the ternary operator, but since that ended up not working out, I'm wondering with keeping the code consistent is more important than trying to express the intent that we're protecting against "undefined". 

I think this expresses the intent better: `json.horizontal_accuracy = this._horizontalAccuracy === undefined ? null : this._horizontalAccuracy;`

But this is shorter, _and_ matches all the other lines: `json.horizontal_accuracy = this._horizontalAccuracy != null ? this._horizontalAccuracy : null;`

Since I have no desire to refactor a bunch of unrelated lines, I'm thinking consistency should win here. Thoughts?

Reply:
For consistency, I think your last solution works. And I did just test it out.

JavaScript falseys can be so unclear.

https://262.ecma-international.org/5.1/#sec-11.9.3

Sometimes null/undefined are the same as 0/false. Sometimes not.